js/controller.js

```diff
@@ -806,6 +813,13 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
 
     onVcPlay: function(event, source) {
       this.state.currentVideoId = source;
+      if (this.state.wasPaused) {
```

Comment:
Should we also check that the previous video was an ad before proceeding inside this if statement?

Reply:
Agree, this `wasPaused` flag is set only when an ad was paused, but I will ad the validation to ensure it was an ssai ad.

Reply:
I will also change the name to `adWasPaused`, to be more clear.